ipn/ipnlocal/local.go

```diff
@@ -6449,13 +6449,35 @@ func (n lastSuggestedExitNode) asAPIType() (res apitype.ExitNodeSuggestionRespon
 	return res, ErrUnableToSuggestLastExitNode
 }
 
+// processAllowedCandidates converts a string slice of exit node candidate IDs to a map of strings and ints. Int value represents nothing and won't be used.
+// Used for a faster lookup comparison in suggestExitNode.
+func processAllowedCandidates(allowedCandidates []string) (candidates set.Set[string]) {
+	for _, candidate := range allowedCandidates {
```

Comment:
@sailorfrag @bradfitz here's how I tried using set. There wasn't much examples of using set.Of so.. got kinda confused

Reply:
that works but isn't taking advantage of the convenience func, see the suggestions